go.mod

```diff
@@ -1,43 +1,41 @@
 module github.com/openshift/oadp-operator
 
-go 1.18
+go 1.20
 
 require (
-	github.com/aws/aws-sdk-go v1.43.31
+	github.com/aws/aws-sdk-go v1.44.253
 	github.com/go-logr/logr v1.2.3
 	github.com/google/uuid v1.3.0
 	github.com/kubernetes-csi/external-snapshotter/client/v4 v4.2.0
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/ginkgo/v2 v2.8.4
 	github.com/onsi/gomega v1.27.2
-	github.com/openshift/api v3.9.0+incompatible
+	github.com/openshift/api v0.0.0-20210805075156-d8fab4513288
```

Comment:
Wanna replace with a newer openshift/api?
https://github.com/openshift/oadp-operator/pull/1088/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6L165

Reply:
The newer one we had in oadp-1.2 pulls in k8s/api 0.26, which I'm trying to avoid. So I reverted to the one we were using prior to our 0.26 change.

Reply:
Updated to the release-4.12 branch (4.13 won't work for us, but 4.12 is fine)